pandas/tests/indexes/multi/test_get_set.py

```diff
@@ -345,6 +345,27 @@ def test_set_names_with_nlevel_1(inplace):
     tm.assert_index_equal(result, expected)
 
 
+def test_multi_set_names_pos_args_deprecation():
+    # GH#41485
+    idx = MultiIndex.from_product([["python", "cobra"], [2018, 2019]])
+
+    msg = (
+        "In a future version of pandas all arguments of Index.set_names "
```

Comment:
same as in the other PR, you'll probably need to define this for `MultiIndex` too (and remove `@final`) so that the warning shows `MultiIndex.set_names`

Reply:
Made this change in fc525c8.